documentation/src/main/asciidoc/topics/ref_infinispan_config.adoc

```diff
@@ -0,0 +1,46 @@
+[id='infinispan-configuration_{context}']
+= Custom {brandname} configuration
```

Comment:
In the context of Infinispan "Custom" doesn't really make sense, as this is simply the Infinispan configuration. "Custom" is an Operator term used to differentiate between user provided configuration and that which is automatically generated by the Operator.